10_datastructures_lab/02.py

```diff
@@ -1,5 +1,26 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+import sys
+from collections import Counter
+print "Please give me a file"
+wordsFile=raw_input()
+words=[]
+counterList=[]
+dict={}
+with open(wordsFile, 'r') as f:
+    for line in f:
+        word=line.strip()
+        words.append(word)
+        cnt=Counter(word)
+        if cnt not in counterList:
+            counterList.append(cnt) 
+
+for cnt in counterList:

```

Comment:
This works but can take a long time if the file is large. For example try to find anagrams in:
https://users.cs.duke.edu/~ola/ap/linuxwords

See if you can find a faster approach that doesn't require a nested loop to print everything